frontend/components/Layout/index.tsx

```diff
@@ -1,3 +1,4 @@
+import Navigation from '../Nav/Nav'
```

Comment:
Just my thoughts, but maybe use a different name for the file? `Nav/Nav` isn't the best option.

Reply:
Can put it `Nav/index` but makes more confusing. So its okay. Since, I've all the component setup like that.

Reply:
Ok.

Reply:
If you name a file index.tsx and import the folder only, i.e., import Navigation from "../Nav". It'll pickup on on the index.ts, so use index.ts (rest of components are like this). This is known as default module name exports. CommonJS looks for index in every folder. Hence why we can create indexes that import/export everything :P

Reply:
@endormi @ale8k no problem. I was creating SEO and layout infront of alex so I may have used index.tsx :P . Willl change the rest to have index.tsx as well.